tests/ProxyManagerTest/Factory/LazyLoadingGhostFactoryTest.php

```diff
@@ -102,6 +102,39 @@ public function testWithOptionalFactory()
     }
 
     /**
+     * @dataProvider skipPropertiesFixture
+     */
+    public function testSkipProperties($className, $properties)
+    {
+        $propertyName = $properties[0]; 
+        $factory      = new LazyLoadingGhostFactory();
+        $ghostObject  = $factory->createProxy(
+            $className,
+            function () use ($propertyName) {
+                throw new \RunTimeException(
+                    sprintf('The property %s not need to be lazy loaded', $propertyName)
+                );
+            },
+            $properties
+        );
+
+        $reflection = new \ReflectionClass($className);
+        $property   = $reflection->getProperty($propertyName);
+        $property->setAccessible(true);
+
+        $this->assertSame($propertyName, $property->getValue($ghostObject));
+    }
+
+    public function skipPropertiesFixture()
+    {
+        return [
+            ['ProxyManagerTestAsset\ClassWithPublicProperties', ['property9']],
```

Comment:
I think the best syntax to use is the internal PHP syntax (`\0ClassName\0privatePropertyName`, `\0*\0protectedPropertyName`, `publicPropertyName`)